test/common/upstream/cluster_manager_impl_test.cc

```diff
@@ -1833,6 +1833,11 @@ TEST_F(ClusterManagerImplTest, DynamicHostRemoveWithTls) {
   ON_CALL(example_com_context, upstreamTransportSocketOptions())
       .WillByDefault(Return(std::make_shared<Network::TransportSocketOptionsImpl>("example.com")));
 
+  NiceMock<MockLoadBalancerContext> example_com_context_with_san;
+  ON_CALL(example_com_context, upstreamTransportSocketOptions())
```

Comment:
example_com_context_with_san ? 

Can you make also use a different name for each one (not example.comf for all) just to make sure there isn't accidental overlap?

Reply:
> Can you make also use a different name for each one (not example.comf for all) just to make sure there isn't accidental overlap?

what do you mean? I specifically made this one `example.com` to test it won't colide with the one without SAN.

Reply:
I meant that you use "example.com" for both SNI and SAN. Can you add a case where they are different and make sure the we get a different conn pool?